jcommune-view/jcommune-web-view/src/test/java/org/jtalks/jcommune/test/AdministrationControllerTest.groovy

```diff
@@ -109,12 +107,117 @@ class AdministrationControllerTest extends Specification {
             def e = thrown(ValidationException)
             [errorMessage] == e.defaultErrorMessages
         and: 'Group is not created'
-            groups.assertDoesNotExist(group)
+            groups.assertDoesNotExistWithName(group.name)
         where:
             groupName           | groupDescription    |errorMessage                                                    | caseName
```

Comment:
Looks like 2 separate tests (for name and for the description). But probably is okay since there is only one check for the description.

These tests should be placed at the unit-test level.